javascript/graphql-sdl-first/README.md

```diff
@@ -33,7 +33,7 @@ Navigate to [http://localhost:4000](http://localhost:4000) in your browser to ex
 
 ## Using the GraphQL API
 
-The schema that specifies the API operations of your GraphQL server is defined in [`./schema.graphql`](./schema.graphql). Below are a number of operations that you can send to the API using the GraphQL Playground.
+The schema that specifies the API operations of your GraphQL server is defined in [`./prisma/schema.graphql`](./schema.graphql). Below are a number of operations that you can send to the API using the GraphQL Playground.
```

Comment:
That file also does not exist in this repo.

Reply:
my bad, maybe the link should just be removed rather than broken

Reply:
I agree - or it should be mentioned that/if the file is generated later or during installation. @nikolasburk